citi-onboarding/Maracatec
# Feature/footer

**What I Did**
-------------
Create Footer page

**How to Test**
--------------
Open the terminal and go to the folder of this project

```
In terminal:

i. git checkout develop
ii. git pull
iii. git checkout feature/Footer
iv. git pull origin feature/Footer
v. open the folder server with comand cd client/
vi. npm -g install yarn
vii. yarn install
vii. yarn start

```
----------------------------------------------------------------
**Check if the component looks just like the mockup in different screen sizes**

**Read the changed files section on GitHub**

**Test the things you think are worth testing, even the ones that are not in this description**
